app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SessionsController < ApplicationController
+  # GET /login
+  def new; end
+
+  # POST /login
+  def create
+    strong_params = session_params
+
+    user = User.find_by(email: strong_params[:email])
+    if !!user&.authenticate(strong_params[:password])
```

Comment:
`session_params` -- this should match your method name `strong_params` is the pattern